src/main/deploy/pathplanner/New Path.path

```diff
@@ -0,0 +1,124 @@
+{
```

Comment:
Don't know how to comment on the file name but will the space cause issues? (Also ideally have a more descriptive name)